redaxo/src/addons/structure/plugins/content/lib/content_service.php

```diff
@@ -160,6 +161,7 @@ public static function copyContent($from_id, $to_id, $from_clang = 1, $to_clang
 
                 $ins->addGlobalUpdateFields();
                 $ins->addGlobalCreateFields();
+                $ins->setTable(rex::getTablePrefix() . 'article_slice');
```

Comment:
bei `->flush()`, und die Methode wird in `execute` aufgerufen, also bei jeder Abfrage.
Hat mich schon öfters gestört, ist aber in R4 genauso.